vp/node/views.py

```diff
@@ -219,6 +219,16 @@ def execute_node(request, node_id):
     # Execute node
     try:
         node_to_execute.execute()
+
+        print(node_to_execute.data)
+
+        # save node_to_execute to a file
+        # TODO: currently hard coding the name of the file as workflow + node id;
+        #       will need to change the word workflow for the workflow name
+        file_name = 'workflow-'+str(node_id)
```

Comment:
Should probably use the FileStorage API

Reply:
There's some setup for the FileStorage in `workflow/views.py`. Pretty sure you could just copy the 
```python
fs = FileSystemStorage(location=settings.MEDIA_ROOT)
```

line into `node/views.py` and then use `fs.open()` and the like. Not sure if we could/should declare the FileStorage centrally, or if it needs to be declared in each module that needs it. I didn't see anything in the documentation about best practices for that.

Reply:
Maybe add this to the TODO, but should we be concerning ourselves with file extensions? Will data always be stored/loaded as JSON in nodes that are not Read/WriteCsv? Even for those nodes, are we saying the data file (i.e. not the input/output CSV) is stored in JSON? My guess is yes, but curious to hear your thoughts.

Reply:
Added a TODO to use FileStorage API. @reelmatt I was thinking that yes, we would store in json. We can see if this posses any issues when we implement those other nodes. At that point we might look into other formats if necessary.